assets/ios/swift/CompoundDesignTokens.swift

```diff
@@ -22,8 +22,8 @@ public class CompoundDesignTokens {
     public static let fontLineHeightBodyMdRegular = CGFloat(20)
     public static let fontLineHeightBodySmRegular = CGFloat(18)
     public static let fontLineHeightBodyXsRegular = CGFloat(16)
-    public static let fontWeightBold = Font.Weight.regular
-    public static let fontWeightSemibold = Font.Weight.regular
+    public static let fontWeightBold = Font.Weight.bold
+    public static let fontWeightSemibold = Font.Weight.semibold
```

Comment:
What happened here? 👀

Reply:
That certainly looks more correct 😅 - we don't use any of the font tokens as they're based on the system ones so we have a manual mapping directly to those.

Reply:
Yeah, I'm really not sure, it looks like the values had been wrong ever since this file was first generated nearly a year ago. It's got to be a bug in Style Dictionary or Tokens Studio Transforms that was fixed in the new releases.